examples/serve/multi_node/multi_node.yaml

```diff
@@ -0,0 +1,49 @@
+# SkyServe YAML to launch a service where each replica contains multiple nodes.
+# Preemption of one node will cause the replica to be preempted and new replica to be launched.
+
+service:
+  readiness_probe: 
+    path: /v1/models
+    initial_delay_seconds: 6000
+  replicas: 1
+
+envs:
+  MODEL_NAME: meta-llama/Llama-2-70b-hf
+  HF_TOKEN: <your-huggingface-token> # TODO: Replace with huggingface token
+
+num_nodes: 2
+
+resources:
+  cloud: gcp
+  ports: 8000
+  accelerators: A100:2
+  use_spot: true
```

Comment:
if we are using spot, maybe we should add a comment saying the multi-node is gang scheduling? actually I'm not sure if we should use spot here. cc @Michaelvll for a look

Reply:
Both spot and non-spot should be gang-scheduled

Reply:
Yeah, just want to say that spot is more error-prone

Reply:
sg, added a comment at the first line